### DISCUSSION TITLE
SceneViewer does not show banner

### DISCUSSION THREAD

**Discussion by 4external** [google/model-viewer#3632] [Q&A]
### Description

With latest ARcore component from google play (1 dec 2021) 
on Samsung a51 SceneViewer show banner and text, but on Sony XZ Premium I don't see banner.
I reproduce it on https://rune-useful-turtle.glitch.me
I don't see useful info in adb logs.

Is there limitations for displaying banner based on screen size, as for pixels and physical size?

#### Live Demo
https://rune-useful-turtle.glitch.me

### Version
SceneViewer

### OS

- [x ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [x ] SceneViewer
- [ ] QuickLook

**Comment by samaneh-kazemi** [comment]
Hi @4external, thanks for reaching out. Can you please share snapshots of what you see in one and the other?

**Comment by 4external** [comment]
@samaneh-kazemi 

This is from SGS A51

![Screenshot_20220207-151050_Google](https://user-images.githubusercontent.com/35369492/152789015-d9073ea7-7cc4-4078-875c-020809048d08.jpg)


And this from Sony

![Screenshot_20220207-151217](https://user-images.githubusercontent.com/35369492/152789034-b38e0ea9-084c-44cf-b6ff-65faa8cc35b9.png)

**Comment by samaneh-kazemi** [comment]
Hi @4external, thanks for providing the details. @eugenecys will look into this issue.

**Comment by eugenecys** [comment]
Hi @4external, that's certainly unexpected. It's most likely due to the screen size. There has been quite a few bug fixes regarding that in the past few weeks, can you check if that's still the case? If so, can you rotate the screen to landscape and see if the banner shows?

Thanks!

**Comment by soruppuAswin** [comment]
I am trying to show few custom glb files into the android app using the SceneViewer. I am following the documentation specified in the [documentation](https://developers.google.com/ar/develop/scene-viewer). Following is the code snipped. 

`val intentUri = Uri.parse("https://arvr.google.com/scene-viewer/1.2").buildUpon()
            .appendQueryParameter(
                "file",
                "https://......./example1.glb"
            )
            .appendQueryParameter("mode", "ar_only")
            .build()`

I am seeing the following error message when I try to view it from SceneViewer. Am I doing anything wrong?
<img width="300" alt="Screen Shot 2022-03-08 at 6 22 11 PM" src="https://user-images.githubusercontent.com/46235940/157342675-e7e18838-3baf-4751-a889-ca9207de142d.png">
